simphony/cuds/mesh.py

```diff
@@ -676,6 +684,33 @@ def has_cells(self):
         """
         return len(self._cells) > 0
 
+    def count_of(self, item_type):
+        """ Return the count of item_type in the container.
+
+        Parameter
+        ---------
+        item_type : CUBA
+            The CUBA enum of the type of the items to return the count of.
+
+        Returns
+        -------
+        count : int
+            The number of items of item_type in the container.
+
+        Raises
+        ------
+        ValueError :
+            If the type of the item is not supported in the current
+            container.
+
+        """
+
+        try:
+            return len(self._items_count[item_type]())
+        except:
```

Comment:
> I wasn't sure if this is a common practice in python. No problem then.

To be honest it all depends on the specifics of the code. Exception handling is always difficult and you can find arguments for both using `try..except` and defensive programming. If for example the operation in the lambda function  could return also a `KeyError` for some other reason then the `try..except` operation would not be wrong but it would be misleading.
The only thing that is always wrong is the simple `except:`.